CRADLE/CallPeak/callPeak.py

```diff
@@ -86,80 +98,69 @@ def mergePeaks(peakResult):
 			qvalues.extend([ currqvalue ])
 		else:
 			## update the continuous regions
-			minPValue = np.min(pvalues)
-			if(minPValue == 0):
-				mergedPeak[resultIdx][4] = np.nan
-			else:
-				mergedPeak[resultIdx][4] = np.round((-1) * np.log10(minPValue), 2)
-
-			minQValue = np.min(qvalues)
-			if(minQValue == 0):
-				mergedPeak[resultIdx][5] = np.nan
-			else:
-				mergedPeak[resultIdx][5] = np.round((-1) * np.log10(minQValue), 2)
+			mergedPeak[resultIdx][4] = updatePvalue(pvalues)
+			mergedPeak[resultIdx][5] = updateQvalue(qvalues)
```

Comment:
For context, see later comment about these two functions
```suggestion
			mergedPeak[resultIdx][4] = newFC(pvalues)
			mergedPeak[resultIdx][5] = newFC(qvalues)
```

Reply:
```suggestion
			mergedPeak[resultIdx][4] = takeMinusLog(pvalues)
			mergedPeak[resultIdx][5] = takeMinusLog(qvalues)
```